packages/shared/src/components/comments/CommentBox.tsx

```diff
@@ -1,168 +1,35 @@
-          : 'border-theme-divider-tertiary',
-        className.container,
-      )}
-      data-testid="comment"
-    >
-      {linkToComment && (
-        <Link href={comment.permalink} prefetch={false} passHref>
-          <CardLink />
-        </Link>
-      )}
-      {children}
-      {showContextHeader && (
-        <header className="mb-4 line-clamp-1 text-text-tertiary typo-footnote">
-          {comment.parent ? (
-            <p>
-              Replied to <strong>@{comment.parent.author.username}</strong>
-            </p>
-          ) : (
-            <p>
-              Commented on <strong>{comment.post.title}</strong>
-            </p>
-          )}
-        </header>
-      )}
-      <header className="z-1 flex w-full flex-row self-start">
-        <ProfileTooltip
-          user={comment.author}
-          tooltip={{ appendTo: appendTooltipTo }}
-        >
-          <ProfileImageLink user={comment.author} />
-        </ProfileTooltip>
-        <div className="ml-3 flex min-w-0 flex-1 flex-col typo-callout">
-          <FlexRow>
-            <CommentAuthor
-              author={comment.author}
-              appendTooltipTo={appendTooltipTo}
-            />
-            <ReputationUserBadge
-              key="reputation"
-              className="ml-1"
-              user={comment.author}
-            />
-            <SquadMemberBadge key="squadMemberRole" role={role} />
-            {comment.author.id === postAuthorId && (
-              <UserBadge
-                key="author"
-                className="text-theme-status-help"
-                content="Creator"
-                Icon={FeatherIcon}
-                iconProps={{
-                  secondary: true,
-                }}
-              />
-            )}
-            {comment.author.id === postScoutId && (
-              <UserBadge
-                key="scout"
-                className="text-theme-color-bun"
-                content="Scout"
-                Icon={ScoutIcon}
-                iconProps={{
-                  secondary: true,
-                }}
-              />
-            )}
-          </FlexRow>
-          <FlexRow className="items-center text-theme-label-quaternary">
-            <ProfileLink href={comment.author.permalink}>
-              <TruncateText title={`@${comment.author.username}`}>
-                @{comment.author.username}
-              </TruncateText>
-            </ProfileLink>
-            <div className="mx-2 h-0.5 w-0.5 bg-theme-label-quaternary" />
-            <CommentPublishDate comment={comment} />
-          </FlexRow>
-        </div>
-      </header>
-      <div
-        className={classNames(
-          'break-words-overflow z-1 mt-3 typo-body',
-          className.content,
-          linkToComment && 'pointer-events-none',
-        )}
-      >
-        <Markdown
-          content={comment.contentHtml}
-          appendTooltipTo={appendTooltipTo}
-        />
+    <CommentContainer
```

Comment:
Most of the content from this file was refactored to `CommentContainer.tsx`. The only difference is, that `CommentContainer` accepts an `actions` prop to render the actions on the bottom. This can be empty for the comment preview for example.